[2024-03-04T20:50:19.502+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados manual__2024-03-04T20:50:12.592306+00:00 [queued]>
[2024-03-04T20:50:19.510+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados manual__2024-03-04T20:50:12.592306+00:00 [queued]>
[2024-03-04T20:50:19.511+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-04T20:50:19.548+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Carregamento_dos_dados> on 2024-03-04 20:50:12.592306+00:00
[2024-03-04T20:50:19.553+0000] {standard_task_runner.py:60} INFO - Started process 1113 to run task
[2024-03-04T20:50:19.556+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Extraçao_de_dados_do_linkedin', 'Carregamento_dos_dados', 'manual__2024-03-04T20:50:12.592306+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/dag_exemplo.py', '--cfg-path', '/tmp/tmpps790hhh']
[2024-03-04T20:50:19.558+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask Carregamento_dos_dados
[2024-03-04T20:50:19.622+0000] {task_command.py:423} INFO - Running <TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados manual__2024-03-04T20:50:12.592306+00:00 [running]> on host 43d6e1a41402
[2024-03-04T20:50:19.868+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Extraçao_de_dados_do_linkedin' AIRFLOW_CTX_TASK_ID='Carregamento_dos_dados' AIRFLOW_CTX_EXECUTION_DATE='2024-03-04T20:50:12.592306+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-04T20:50:12.592306+00:00'
[2024-03-04T20:50:20.793+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-04T20:50:20.804+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Extraçao_de_dados_do_linkedin, task_id=Carregamento_dos_dados, execution_date=20240304T205012, start_date=20240304T205019, end_date=20240304T205020
[2024-03-04T20:50:20.852+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-04T20:50:20.866+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
